src/material.js

```diff
@@ -172,6 +172,19 @@ export class Material {
         return new Lattice(this.lattice);
     }
 
+    getDerivedPropertiesForHash() {
+        const derivedProperties = this._json.derivedProperties;
+        if (derivedProperties) {
```

Comment:
We only want to run the code below if `derivedProperties` has been created and filled with properties. Otherwise the Error will get thrown because an empty array leads to `!inchi`, which triggers the error.